hibernate-core/src/main/java/org/hibernate/loader/entity/EntityLoader.java

```diff
@@ -140,14 +139,6 @@ public EntityLoader(
 		}
 	}
 
-	public Object loadByUniqueKey(SharedSessionContractImplementor session, Object key) {
-		return loadByUniqueKey( session, key, null );
-	}
-
-	public Object loadByUniqueKey(SharedSessionContractImplementor session, Object key, Boolean readOnly) {
-		return load( session, key, null, null, LockOptions.NONE, readOnly );
-	}
-
```

Comment:
These really weren't doing a lot of work, and HR can't override them.

Reply:
I'll change this to deprecate them, suggesting to use `load` instead. Ok?

Reply:
Totally fine by me :)